testdata/p4_16_samples_outputs/default_action-bmv2.p4.p4info.txtpb

```diff
@@ -14,6 +14,13 @@ tables {
     id: 29022044
   }
   const_default_action_id: 29022044
+  initial_default_action {
+    action_id: 29022044
+    arguments {
+      param_id: 1
+      value: "\000\000\000\n"
```

Comment:
I am not sure, but I thought that perhaps P4Runtime action parameters were intended to be specified without leading 0s, i.e. a variable-length encoding that has at least one byte, but can have fewer than the number of bytes implied by the bitwidth, because leading 0 bytes are omitted?

Reply:
@chrispsommers @smolkaj Do you recall the answer to my question above?

Reply:
I am using the control-plane `stringRepr` function, which is also used to generate entries.
There are some notes: https://github.com/p4lang/p4c/blob/main/control-plane/bytestrings.cpp#L29

Reply:
Yeah, part of my confusion is that I recall this coming up in conversations since around 2019, and I never mentally keep track whether this has been changed.  I guess all pieces of software involved will probably forever either support the non-canonical full bit width values, and/or perhaps some day transition to the canonical variable-length ones.  No need to tackle that issue in this commit.